test/image/mocks/heatmap_multi-trace.json

```diff
@@ -0,0 +1,193 @@
+{
+  "data": [
+    {
+      "x": [
```

Comment:
We didn't have a mock for a subplot with multiple heatmap traces on it yet. We have numerous multiple subplots mocks with one heatmap per subplot, but none for multiple heatmap on the same x/y subplot. So, here's one.